src/views/Home/components/Hero/Hero.scss

```diff
@@ -25,8 +48,107 @@
     }
   }
 
+  // background
+  &__grid {
+    position: absolute;
+    transform: scaleY(-1);
+    z-index: -2;
+    bottom: 0;
+    left: 6vw;
+    max-width: calc(90% - 40vw);
+    max-height: 50%;
+    min-height: calc(20em + 60vh);
+
+    @include responsiveness.respond(tab-port) {
+      left: 0;
+      max-width: calc(20vw + 15vh);
+      min-height: 90%;
+      max-height: 100%;
+    }
+
+    @include responsiveness.respond(smallest) {
+      visibility: hidden;
+    }
+  }
```

Comment:
Reference my previous comment about simiplying this structure with a single `*.svg` file.

Reply:
responsiveness is more flexible using this stylesheet structure! check out how it responds across ultrawide to mobile

Reply:
Acknowledged!